execution/executor/tests/storage_integration_test.rs

```diff
@@ -241,6 +241,226 @@ fn test_reconfiguration() {
     // validator set fails to parse
 }
 
+#[test]
```

Comment:
We should probably have a dedicated directory for testing end to end between transactions, storage, and VM and probably outside the scope of the executor itself. Since these aren't really testing the executor.

Reply:
Agreed. `libra/testsuite/tests/libratest` might be a good candidate. It already has a few tests that fit this description

Some of the tests in `libra/language/e2e-tests` could probably move to this new dir as well.